README.md

```diff
@@ -117,6 +117,16 @@ It is also possible to set the latest JFrog CLI version by adding the *version*
 | Important: Only JFrog CLI versions 1.29.0 or above are supported. |
 | --- |
 
+## Downloading JFrog CLI From JFrog Artifactory
+If your agent has no internet access, you can configure the workflow to download JFrog CLI from a JFrog Artifactory instance, which is configured to proxy the download repository:
```

Comment:
...repository: --> ...repository.

Let add the following in a new line right after this line - 
```Here's how you do it:```